package/MDAnalysis/coordinates/base.py

```diff
@@ -1293,13 +1294,20 @@ def check_slice_indices(self, start, stop, step):
             range(10)[10:-1:-1]  # []
 
         """
-        for var, varname in (
-                (start, 'start'),
-                (stop, 'stop'),
-                (step, 'step')
-        ):
-            if not (isinstance(var, int) or (var is None)):
+
```

Comment:
If we need this slice processing in more than one location then it should probably be a function in `lib.util` (or similar).